src/FormInterface.php

```diff
@@ -4,21 +4,241 @@
 
 namespace Yiisoft\Form;
 
-interface FormInterface
+use Yiisoft\Validator\DataSetInterface;
+
+interface FormInterface extends DataSetInterface
```

Comment:
Should `validate()` be added to the interface?

Reply:
Done.